gslb/ingestion/bootup_handler.go

```diff
@@ -184,8 +184,8 @@ func HandleBootup(cfg *restclient.Config) (bool, error) {
 	}
 
 	if len(amkoClusterList.Items) == 0 {
-		gslbutils.Logf("No AMKOCluster object found, AMKO would start as leader")
-		return true, nil
+		gslbutils.Logf("No AMKOCluster object found, AMKO would stop")
```

Comment:
Are we fixing another issue as part of this issue? Should we mention that ID to track it?

Reply:
Have added AV-228538 in description. Also present in commit message